sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -6496,4 +6496,12 @@ ALTER TABLE cases_history
 
 INSERT INTO schema_version (version_number, comment) VALUES (326, 'SurvNet Adaptations - Create new field “Reinfection” for cases #3831');
 
+-- 2021-02-05 Add reason hospitalization #4187
+ALTER TABLE hospitalization ADD COLUMN reasonforhospitalization varchar(255);
+ALTER TABLE hospitalization ADD COLUMN otherreasonforhospitalization varchar(255);
```

Comment:
This needs to be of type "text" because it's a free text field - we use varchar(255) for enums, have used varchar(512) for free text fields for a long time, but have decided that "text" is a way better alternative